src/nomad_simulations/schema_packages/numerical_settings.py

```diff
@@ -890,15 +911,295 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        shape=['n_terms'],
+        description="""
+        Allow the code to optimize the initial energy parameter.
+        """,
+    )
+
+    differential_orders = Quantity(
+        type=np.int32,
+        shape=['n_terms'],
+        description="""
+        Derivative order of the radial wavefunction term.
+        """,
+    )  # TODO: add check non-negative # ? to remove
+
+    boundary_orders = Quantity(
+        type=np.int32,
+        shape=['n_terms'],
+        description="""
+        Differential order to which the radial wavefunction is matched at the boundary.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+        if np.any(np.isneginf(self.differential_orders)):
+            logger.error('`APWLOrbital.differential_order` must be non-negative.')
+        if np.any(np.isneginf(self.boundary_orders)):
+            logger.error('`APWLOrbital.boundary_orders` must be non-negative.')
+
+
+class APWLChannel(ArchiveSection):
+    """
+    Channel of the angular momentum ($l$),
+    containing all orbitals that contribute to its description.
+    """
+
+    name = Quantity(
+        type=np.int32,
+        description="""
+        Angular momentum quantum number of the local orbital.
+        """,
+    )
+
+    n_wavefunctions = Quantity(
+        type=np.int32,
+        description="""
+        Number of wavefunctions in the l-channel, i.e. $(2l + 1) n_orbitals$.
+        """,
+    )
+
+    orbitals = SubSection(sub_section=APWWavefunction.m_def, repeats=True)
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+        # self.name = self.m_def.name
+        self.n_wavefunctions = len(self.orbitals) * (2 * self.name + 1)
+
+
+class MuffinTinRegion(LocalizedBasisSet, Mesh):
+    """
+    Muffin-tin region around atoms, containing the augmented part of the APW basis set.
+    """
+
+    # there are 2 main ways of structuring the APW basis set
+    # either as APW and lo in the MT region
+    # or by l-channel in the MT region
+
+    l_max = Quantity(
+        type=np.int32,
+        description="""
+        Maximum angular momentum quantum number that is sampled.
+        Starts at 0.
+        """,
+    )
+
+    l_channels = SubSection(sub_section=APWLChannel.m_def, repeats=True)
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+        self.type = 'Spherical'
+
+
+class BasisSetContainer(NumericalSettings):
+    """Section providing an overview of the full basis set.
+    The basis set may contain multiple basis sets specifications `BasisSet`,
+    each with their own parameters."""
+
+    native_tier = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        Code-specific tag indicating the precision used
```

Comment:
Maybe specify a couple of examples?

Reply:
They're very code-dependent, which I would reserve for the parser plugins.
I added this to the description with a generic example.